### DISCUSSION TITLE
XRechnung validation fails due PartyName

### DISCUSSION THREAD

**Discussion by fisfro** [stephanstapel/ZUGFeRD-csharp#454] [Q&A]
Hello,

I'm completely new to this topic and try to implement this libary in my app, but I have problem with validation. With Profile=XRechnung I will get the supplier name direct in the node PartyName. But according to the validator it should look like
<cac:PartyName>
	 <cbc:Name>Lieferant GmbH</cbc:Name>
	</cac:PartyName>
      <cac:PostalAddress>

I'm using the latest version V15 and I'm not able to figure out , where i can change this setting.
currently I'm using 
 desc.Save(memoryStream, ZUGFeRDVersion.Version23,s2industries.ZUGFeRD.Profile.XRechnung,ZUGFeRDFormats.UBL);

Further I would like to ask, why spain will be not part of the ElectronicAddressSchemeIdentifiers enum? Is there a reason or only missing?

Any help would be welcome.

Robert

**Comment by stephanstapel** [comment]
Hi,

this should not be present in the latest version which you find on github. Please use this code until a new release is created.

Cheers,

Stephan

**Reply by fisfro** [reply]
Hello Stephan,

thanks , that helps.....

CIAO Robert
